### ISSUE TITLE
Processing pages with many different spatial features

### ISSUE THREAD

**Issue by fils** [earthcubearchitecture-project418/crawler#10]
Ref:  http://get.iedadata.org/doi/100700

If you look at the spatial part of this page you will see that there is both a bounding box and points.  The bounding box is a feature around the points.  

I have not yet looked to see how this got processed by our spatial indexer, but I will.

**Comment by fils** [comment]
Not seeing this item in the spatial index

```
127.0.0.1:9851> get p418 http://get.iedadata.org/doi/100700
(error) id not found
127.0.0.1:9851> get p418 http://get.iedadata.org/doi/100301
{"ok":true,"object":{"type":"Polygon","coordinates":[[[-58.315452,-62.245245],[-58.315452,-60.928837],[-56.29236,-60.928837],[-56.29236,-62.245245],[-58.315452,-62.245245]]]},"elapsed":"62.978µs"}
127.0.0.1:9851> 
```

**Comment by ashepherd** [comment]
Looks like the frame is working: 

[JSON-LD Playground](https://json-ld.org/playground/#startTab=tab-framed&json-ld=%7B%22%40context%22%3A%7B%22%40vocab%22%3A%22http%3A%2F%2Fschema.org%2F%22%2C%22datacite%22%3A%22http%3A%2F%2Fpurl.org%2Fspar%2Fdatacite%2F%22%2C%22earthcollab%22%3A%22https%3A%2F%2Flibrary.ucar.edu%2Fearthcollab%2Fschema%23%22%2C%22geolink%22%3A%22http%3A%2F%2Fschema.geolink.org%2F1.0%2Fbase%2Fmain%23%22%2C%22vivo%22%3A%22http%3A%2F%2Fvivoweb.org%2Fontology%2Fcore%23%22%2C%22dcat%22%3A%22http%3A%2F%2Fwww.w3.org%2Fns%2Fdcat%23%22%7D%2C%22%40id%22%3A%22%20DOI%3A10.1594%2FIEDA%2F100700%22%2C%22%40type%22%3A%22Dataset%22%2C%22additionalType%22%3A%5B%22geolink%3ADataset%22%2C%22vivo%3ADataset%22%5D%2C%22name%22%3A%22S%2C%20Se%20and%20Te%20contents%20of%20basalts%20from%20the%20Reykjanes%20Ridge%20and%20SW%20Iceland%20Rift%20Zone%22%2C%22citation%22%3A%22Allison%20Forrest%2C%20Jean-Guy%20Schilling%2C%20Katherine%20A.%20Kelley%20(2017)%2C%20S%2C%20Se%20and%20Te%20contents%20of%20basalts%20from%20the%20Reykjanes%20Ridge%20and%20SW%20Iceland%20Rift%20Zone.%20Interdisciplinary%20Earth%20Data%20Alliance%20(IEDA).%20doi%3A10.1594%2FIEDA%2F100700%22%2C%22creator%22%3A%5B%7B%22%40type%22%3A%22Person%22%2C%22additionalType%22%3A%22geolink%3APerson%22%2C%22name%22%3A%22Allison%20Forrest%22%2C%22givenName%22%3A%22Allison%22%2C%22familyName%22%3A%22Forrest%22%7D%2C%7B%22%40type%22%3A%22Person%22%2C%22additionalType%22%3A%22geolink%3APerson%22%2C%22name%22%3A%22Jean-Guy%20Schilling%22%2C%22givenName%22%3A%22Jean-Guy%22%2C%22familyName%22%3A%22Schilling%22%7D%2C%7B%22%40type%22%3A%22Person%22%2C%22additionalType%22%3A%22geolink%3APerson%22%2C%22name%22%3A%22Katherine%20A.%20Kelley%22%2C%22givenName%22%3A%22Katherine%22%2C%22familyName%22%3A%22Kelley%22%7D%5D%2C%22datePublished%22%3A%222017%22%2C%22dateCreated%22%3A%222017-06-28%22%2C%22version%22%3A%221.0%22%2C%22inLanguage%22%3A%22en%22%2C%22description%22%3A%22Abstract%3A%20This%20data%20set%20reports%20S%2C%20Se%2C%20and%20Te%20concentrations%20in%20basaltic%20whole-rocks%20and%20glasses%20from%20the%20SW%20Iceland%20rift%20zone%2C%20including%20sub-glacial%20glasses%2C%20and%20submarine%20lavas%20from%20the%20Reykjanes%20Ridge%2C%20Gibbs%20Fracture%20Zone%2C%20and%20Mid-Atlantic%20Ridge.%20We%20thank%20R.%20Kingsley%2C%20J.M.%20Rhodes%2C%20and%20C.%20Mandeville%20for%20assistance%20with%20analyses.%20This%20work%20was%20supported%20by%20NSF%20award%20OCE%23%200326658%20to%20JGS.%20NSF%20award%20OCE%23%201555523%20provides%20curatorial%20support%20for%20marine%20geological%20samples%20at%20the%20Graduate%20School%20of%20Oceanography%2C%20University%20of%20Rhode%20Island.%3B%20Other%20Description%3A%20Forrest%2C%20A.%20(2005)%2C%20Sulfur%2C%20Selenium%20and%20Tellurium%20in%20Reykjanes%20Ridge%20and%20Iceland%20Basalts%2C%20175%20pp%2C%20University%20of%20Rhode%20Island%2C%20Kingston%2C%20RI.%20Forrest%2C%20A.%2C%20R.%20Kingsley%2C%20and%20J.-G.%20Schilling%20(2009)%2C%20Determination%20of%20Selenium%20and%20Tellurium%20in%20Basalt%20Rock%20Reference%20Materials%20by%20Isotope%20Dilution%20Hydride%20Generation-Inductively%20Coupled%20Plasma-Mass%20Spectrometry%20(ID-HG-ICP-MS)%2C%20Geostandards%20and%20Geoanalytical%20Research%2C%2033(2)%2C%20261-269%2C%20doi%3A10.1111%2Fj.1751-908X.2009.00841.x.%22%2C%22distribution%22%3A%5B%7B%22%40type%22%3A%22DataDownload%22%2C%22additionalType%22%3A%22dcat%3Adistribution%22%2C%22name%22%3A%22DOI%20landing%20page%22%2C%22dcat%3AaccessURL%22%3A%22http%3A%2F%2Fdx.doi.org%2F10.1594%2FIEDA%2F100700%22%2C%22url%22%3A%22http%3A%2F%2Fdx.doi.org%2F10.1594%2FIEDA%2F100700%22%2C%22encodingFormat%22%3A%5B%22application%2Fpdf%22%2C%22application%2Fvnd.ms-excel%22%5D%7D%2C%7B%22%40type%22%3A%22DataDownload%22%2C%22additionalType%22%3A%22dcat%3Adistribution%22%2C%22name%22%3A%22URL%22%2C%22dcat%3AaccessURL%22%3A%22http%3A%2F%2Fwww.earthchem.org%2Flibrary%2Fbrowse%2Fview%3Fid%3D1085%22%2C%22url%22%3A%22http%3A%2F%2Fwww.earthchem.org%2Flibrary%2Fbrowse%2Fview%3Fid%3D1085%22%2C%22encodingFormat%22%3A%5B%22application%2Fpdf%22%2C%22application%2Fvnd.ms-excel%22%5D%7D%5D%2C%22identifier%22%3A%5B%7B%22%40id%22%3A%22%20doi%3A10.1594%2FIEDA%2F100700%22%2C%22%40type%22%3A%22PropertyValue%22%2C%22additionalType%22%3A%5B%22geolink%3AIdentifier%22%2C%22datacite%3AIdentifier%22%5D%2C%22propertyID%22%3A%22datacite%3Adoi%22%2C%22url%22%3A%22http%3A%2F%2Fdx.doi.org%2F10.1594%2FIEDA%2F100700%22%2C%22value%22%3A%2210.1594%2FIEDA%2F100700%22%7D%5D%2C%22isPartOf%22%3A%5B%7B%22%40id%22%3A%22http%3A%2F%2Fwww.nsf.gov%2Fawardsearch%2FshowAward.do%3FAwardNumber%3D0326658%22%2C%22%40type%22%3A%22CreativeWork%22%2C%22additionalType%22%3A%22earthcollab%3AProject%22%2C%22funder%22%3A%7B%22%40type%22%3A%22Organization%22%2C%22name%22%3A%22National%20Science%20Foundation%22%7D%7D%2C%7B%22%40id%22%3A%22http%3A%2F%2Fwww.nsf.gov%2Fawardsearch%2FshowAward.do%3FAwardNumber%3D1555523%22%2C%22%40type%22%3A%22CreativeWork%22%2C%22additionalType%22%3A%22earthcollab%3AProject%22%2C%22funder%22%3A%7B%22%40type%22%3A%22Organization%22%2C%22name%22%3A%22National%20Science%20Foundation%22%7D%7D%5D%2C%22keywords%22%3A%5B%22Regional%20(Continents%2C%20Oceans)%22%2C%22Spreading%20Center%22%2C%22Chemistry%3ARock%22%2C%22Geochemistry%22%2C%22Marine%20Geoscience%22%2C%22Solid%20Earth%22%2C%22basalt%22%2C%22mid-ocean%20ridge%22%2C%22plume%22%2C%22Selenium%22%2C%22sulfur%22%2C%22Tellurium%22%2C%22Gibbs%20Fracture%20Zone%22%2C%22Iceland%22%2C%22Mid-Atlantic%20Ridge%22%2C%22Reykjanes%20Ridge%22%2C%22Reykjanes%20Ridge%2C%20Iceland%2C%20Mid-Atlantic%20Ridge%2C%20Gibbs%20Fracture%20Zone%22%5D%2C%22license%22%3A%22Creative%20Commons%20Attribution-NonCommercial-Share%20Alike%203.0%20United%20States%20%5BCC%20BY-NC-SA%203.0%5D%22%2C%22provider%22%3A%7B%22%40type%22%3A%22Organization%22%2C%22%40id%22%3A%22https%3A%2F%2Fwww.iedadata.org%2F%22%2C%22name%22%3A%22Interdisciplinary%20Earth%20Data%20Alliance%20(IEDA)%22%7D%2C%22publisher%22%3A%7B%22%40type%22%3A%22Organization%22%2C%22%40id%22%3A%22https%3A%2F%2Fwww.iedadata.org%2F%22%2C%22name%22%3A%22Interdisciplinary%20Earth%20Data%20Alliance%20(IEDA)%22%2C%22url%22%3A%22https%3A%2F%2Fwww.iedadata.org%2F%22%2C%22description%22%3A%22The%20IEDA%20data%20facility%20mission%20is%20to%20support%2C%20sustain%2C%20and%20advance%20the%20geosciences%20by%20providing%20data%20services%20for%20observational%20geoscience%20data%20from%20the%20Ocean%2C%20Earth%2C%20and%20Polar%20Sciences.%20IEDA%20systems%20serve%20as%20primary%20community%20data%20collections%20for%20global%20geochemistry%20and%20marine%20geoscience%20research%20and%20support%20the%20preservation%2C%20discovery%2C%20retrieval%2C%20and%20analysis%20of%20a%20wide%20range%20of%20observational%20field%20and%20analytical%20data%20types.%20Our%20tools%20and%20services%20are%20designed%20to%20facilitate%20data%20discovery%20and%20reuse%20for%20focused%20disciplinary%20research%20and%20to%20support%20interdisciplinary%20research%20and%20data%20integration.%22%2C%22logo%22%3A%7B%22%40type%22%3A%22ImageObject%22%2C%22url%22%3A%22http%3A%2F%2Fapp.iedadata.org%2Fimages%2Fieda_maplogo.png%22%7D%2C%22contactPoint%22%3A%7B%22%40type%22%3A%22ContactPoint%22%2C%22name%22%3A%22Information%20Desk%22%2C%22email%22%3A%22info%40iedadata.org%22%2C%22url%22%3A%22https%3A%2F%2Fwww.iedadata.org%2Fcontact%2F%22%2C%22contactType%22%3A%22Information%22%7D%2C%22parentOrganization%22%3A%7B%22%40type%22%3A%22Organization%22%2C%22%40id%22%3A%22https%3A%2F%2Fviaf.org%2Fviaf%2F142992181%2F%22%2C%22name%22%3A%22Lamont-Doherty%20Earth%20Observatory%22%2C%22url%22%3A%22http%3A%2F%2Fwww.ldeo.columbia.edu%2F%22%2C%22address%22%3A%7B%22%40type%22%3A%22PostalAddress%22%2C%22streetAddress%22%3A%2261%20Route%209W%22%2C%22addressLocality%22%3A%22Palisades%22%2C%22addressRegion%22%3A%22NY%22%2C%22postalCode%22%3A%2210964-1000%22%2C%22addressCountry%22%3A%22USA%22%7D%2C%22parentOrganization%22%3A%7B%22%40type%22%3A%22Organization%22%2C%22%40id%22%3A%22https%3A%2F%2Fviaf.org%2Fviaf%2F156836332%2F%22%2C%22legalName%22%3A%22Columbia%20University%22%2C%22url%22%3A%22http%3A%2F%2Fwww.columbia.edu%2F%22%7D%7D%2C%22funder%22%3A%7B%22%40type%22%3A%22Organization%22%2C%22%40id%22%3A%22http%3A%2F%2Fdx.doi.org%2F10.13039%2F100000085%22%2C%22legalName%22%3A%22Directorate%20for%20Geosciences%22%2C%22alternateName%22%3A%22NSF-GEO%22%2C%22url%22%3A%22http%3A%2F%2Fwww.nsf.gov%22%2C%22parentOrganization%22%3A%7B%22%40type%22%3A%22Organization%22%2C%22%40id%22%3A%22http%3A%2F%2Fdx.doi.org%2F10.13039%2F100000001%22%2C%22legalName%22%3A%22National%20Science%20Foundation%22%2C%22alternateName%22%3A%22NSF%22%2C%22url%22%3A%22http%3A%2F%2Fwww.nsf.gov%22%7D%7D%2C%22publishingPrinciples%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fcreativecommons.org%2Flicenses%2Fby-nc-sa%2F3.0%2Fus%2F%22%2C%22%40type%22%3A%22DigitalDocument%22%2C%22additionalType%22%3A%22gdx%3AProtocol-License%22%2C%22name%22%3A%22Dataset%20Usage%20License%22%2C%22description%22%3A%22Creative%20Commons%20Attribution-NonCommercial-Share%20Alike%203.0%20United%20States%20%5BCC%20BY-NC-SA%203.0%5D%22%2C%22url%22%3A%22https%3A%2F%2Fcreativecommons.org%2Flicenses%2Fby-nc-sa%2F3.0%2Fus%2F%22%7D%7D%2C%22spatialCoverage%22%3A%5B%7B%22%40type%22%3A%22Place%22%2C%22name%22%3A%22Reykjanes%20Ridge%2C%20Iceland%2C%20Mid-Atlantic%20Ridge%2C%20Gibbs%20Fracture%20Zone%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoShape%22%2C%22box%22%3A%22-35.4%2C%2050.46%20-17.68%2C%2065%22%7D%2C%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-29.42%22%2C%22latitude%22%3A%2250.46%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-30.02%22%2C%22latitude%22%3A%2251.28%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-29.92%22%2C%22latitude%22%3A%2251.56%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-29.95%22%2C%22latitude%22%3A%2252.01%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-31.52%22%2C%22latitude%22%3A%2252.33%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-31.57%22%2C%22latitude%22%3A%2252.48%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-34.94%22%2C%22latitude%22%3A%2252.66%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-35.24%22%2C%22latitude%22%3A%2253.41%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-35.4%22%2C%22latitude%22%3A%2254.25%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-35.22%22%2C%22latitude%22%3A%2254.76%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-34.87%22%2C%22latitude%22%3A%2255.67%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-33.42%22%2C%22latitude%22%3A%2257.14%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-32.57%22%2C%22latitude%22%3A%2257.68%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-31.62%22%2C%22latitude%22%3A%2258.42%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-30.95%22%2C%22latitude%22%3A%2258.87%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-29.53%22%2C%22latitude%22%3A%2259.99%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-29.51%22%2C%22latitude%22%3A%2259.99%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-29.43%22%2C%22latitude%22%3A%2259.99%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-29.48%22%2C%22latitude%22%3A%2260.02%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-29.38%22%2C%22latitude%22%3A%2260.03%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-28.88%22%2C%22latitude%22%3A%2260.45%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-28.42%22%2C%22latitude%22%3A%2260.73%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-27.9%22%2C%22latitude%22%3A%2261.09%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-27.88%22%2C%22latitude%22%3A%2261.1%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-27.07%22%2C%22latitude%22%3A%2261.6%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-26.88%22%2C%22latitude%22%3A%2261.73%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-26.54%22%2C%22latitude%22%3A%2261.98%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-26.29%22%2C%22latitude%22%3A%2262.08%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-26.36%22%2C%22latitude%22%3A%2262.11%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-26.14%22%2C%22latitude%22%3A%2262.26%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-25.93%22%2C%22latitude%22%3A%2262.31%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-25.78%22%2C%22latitude%22%3A%2262.35%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-25.84%22%2C%22latitude%22%3A%2262.37%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-25.45%22%2C%22latitude%22%3A%2262.59%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-25.43%22%2C%22latitude%22%3A%2262.62%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-25.21%22%2C%22latitude%22%3A%2262.7%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-25.16%22%2C%22latitude%22%3A%2262.79%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-24.88%22%2C%22latitude%22%3A%2262.9%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-24.69%22%2C%22latitude%22%3A%2262.99%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-24.5%22%2C%22latitude%22%3A%2263.07%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-24.46%22%2C%22latitude%22%3A%2263.19%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-24.27%22%2C%22latitude%22%3A%2263.22%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-24.2%22%2C%22latitude%22%3A%2263.27%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-24.243%22%2C%22latitude%22%3A%2263.277%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-23.85%22%2C%22latitude%22%3A%2263.46%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-23.87%22%2C%22latitude%22%3A%2263.47%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-23.8%22%2C%22latitude%22%3A%2263.47%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-23.7%22%2C%22latitude%22%3A%2263.57%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-22.7%22%2C%22latitude%22%3A%2263.85%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-22.43%22%2C%22latitude%22%3A%2263.86%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-22.05%22%2C%22latitude%22%3A%2263.9%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-22.54%22%2C%22latitude%22%3A%2263.91%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-22.2%22%2C%22latitude%22%3A%2263.95%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-21.74%22%2C%22latitude%22%3A%2263.97%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-22.57%22%2C%22latitude%22%3A%2263.98%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-21.97%22%2C%22latitude%22%3A%2263.98%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-21.9%22%2C%22latitude%22%3A%2264.1%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-18.99%22%2C%22latitude%22%3A%2264.23%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-20.435%22%2C%22latitude%22%3A%2264.277%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-21.11%22%2C%22latitude%22%3A%2264.28%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-21%22%2C%22latitude%22%3A%2264.33%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-21.08%22%2C%22latitude%22%3A%2264.45%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-19.155%22%2C%22latitude%22%3A%2264.663%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-17.68%22%2C%22latitude%22%3A%2264.68%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-20.78%22%2C%22latitude%22%3A%2264.73%22%7D%5D%7D%2C%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%5B%7B%22%40type%22%3A%22GeoCoordinates%22%2C%22longitude%22%3A%22-19.58%22%2C%22latitude%22%3A%2265%22%7D%5D%7D%5D%7D&frame=%7B%22%40context%22%3A%22http%3A%2F%2Fschema.org%2F%22%2C%22%40explicit%22%3Atrue%2C%22%40type%22%3A%22Dataset%22%2C%22%40id%22%3A%22%22%2C%22spatialCoverage%22%3A%7B%22%40type%22%3A%22Place%22%2C%22geo%22%3A%7B%7D%2C%22subjectOf%22%3A%7B%22%40type%22%3A%22CreativeWork%22%2C%22%40explicit%22%3Atrue%2C%22fileFormat%22%3A%22%22%2C%22text%22%3A%22%22%7D%7D%7D)

**Comment by ashepherd** [comment]
working it out here: https://github.com/earthcubearchitecture-project418/crawler/tree/spatial-frame-arrays

Arrays still causing issues even with byte arrays

**Comment by ashepherd** [comment]
closed in pull request #11
